src/js/components/Peak/PeakDescription.tsx

```diff
@@ -24,13 +37,91 @@ const PeakDescription = ({ peak, showForm }: PeakDescriptionProps) => {
       <p>Województwo: {peak.region}</p>
       <p>Pasmo górskie: {peak.mountainRange}</p>
       <PeakStatistics peakId={peak.id} />
-      {showForm && (
-        <div className={styles.buttonBox}>
-          <SubmitButton
-            onClick={() => reachPeak(3000)}
-            text="Zaznacz jako zdobyty"
-            className={styles.reachPeakButton}
-          />
+      {true && (
```

Comment:
tu będzie `showForm` ale na razie jest true, trzeba będzie to przedyskutować, bo to w sumie chyba nie powinno być tak że tylko raz można sobie wprowadzić czas, bo kilka razy można chodzić na dany szczyt, ta flaga powinna tylko odpowiadać imo za wyświetlanie tej ikonki:
![image](https://user-images.githubusercontent.com/27630111/120110534-28ccb600-c16e-11eb-9a24-f8e2bcd21c08.png)

Reply:
Kwestia jest tylko taka, w jaki sposób pokazać że zdobyliśmy szczyt kilka razy? Natomiast rzeczywiście wydaje mi się że ten modal powinien óc nam wyskoczyć nawet po zdobyciu szczytu

Reply:
Tylko co w takim razie ma być przechowywane w bazie danych? Bo jak wszystkie czasy, to trzeba by zmienić model. Jak tylko najkrótszy czas, to wtedy jakby ktoś zdobył jako pierwszy dany szczyt, a potem zdobył go jeszcze raz tylko szybciej, to już mógłby nie być w statystykach jako pierwszy zdobywca szczytu.

Reply:
Myślę że rozwiązanie tego powinno być takie, że w tabeli ze szczytami pierwszy zdobywca powinien być osobną kolumną, natomiast czas powinien się aktualizować żeby mogł brać udział w rankingu najszybszych zdobywców czy coś takiego.